uniswap_swap_bot/src/bot.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+} from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+
+import bot from "./bot";
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+describe("swap occur", () => {
+  let handleTransaction: HandleTransaction;
+
+  const mockTxEvent = createTransactionEvent({} as any);
+
+  beforeAll(() => {
+    handleTransaction = bot.handleTransaction;
+  });
+
+  describe("token swap", () => {
```

Comment:
To increase code coverage, and after you adjust your logic for what we were looking for, you can add test cases to make sure your bot doesn't trigger for:
- a function call that is NOT a `swap`
- a `swap` that occurred on a contract that is NOT a valid UniswapV3 pool
- confirm it can alert on multiple `swap`s in the same transaction
- correctly alert on valid `swap`s from a valid UniswapV3 pool that is in the same transaction as `swap`s that are NOT from a valid UniswapV3 pool